shell/browser/native_browser_view_views.cc

```diff
@@ -92,9 +92,35 @@ void NativeBrowserViewViews::ResetAutoResizeProportions() {
 }
 
 void NativeBrowserViewViews::SetBounds(const gfx::Rect& bounds) {
+  // Previous implementation
+  /*
   auto* view = GetInspectableWebContentsView()->GetView();
   view->SetBoundsRect(bounds);
   ResetAutoResizeProportions();
+  */
+
+  /*
+  This implementation is attempting to ignore setting bounds
+  if the view has been destroyed, (ie invalid).
+  A different check is most likely needed as the destroyed
+  pointer cannot be guaranteed NULL.
+  When the pointer is NULL, the program does not crash, but when
+  it is not, it proceeds to still crash.
+  */
+  auto* temp = GetInspectableWebContentsView();
+  LOG(INFO) << "NativeBrowserViewViews::SetBounds - temp: " << temp
```

Comment:
We do want to return early if there's no webContents. However, while this crash is specific to `setBounds`, if we take a look around this file, we'll notice that `GetInspectableWebContentsView()` is called in a large handful of other places. As such, any other code invoking those methods after a webContents is destructed will _also_ cause a crash, and we need to check those as well. In this case, `OnWidgetBoundsChanged` is called when the window is destructed during shutdown, and this method invokes `SetAutoResizeProportions`, which in turn makes the (incorrect) assumption that `GetInspectableWebContentsView()` is not a `nullptr`. Guarding against that case will likely fix the problem.

Reply:
There are also some instances of this in the correspondent native_browser_view_mac.mm file which will also need to be checked and handled :)